CHANGELOG.md

```diff
@@ -1,3 +1,29 @@
+# dbt_unified_rag v0.1.0-a7
+[PR #25](https://github.com/fivetran/dbt_unified_rag/pull/25) introduces the following updates that **require a full refresh**.
+
+## Bug Fixes 
+- Fixed an issue in which [unioned](https://github.com/fivetran/dbt_unified_rag?tab=readme-ov-file#union-multiple-connections) source connections were producing null models.
+  - The solution required the addition of a base staging model layer. For each staging model, there is a `*_base` counterpart in which we are running our `union_data` macro. This framework is necessary to the cooperation of our unioning and column-filling macros, which ensure the models do not fail if you are missing an expected column.
+  - For each connector type, this adds:
+    - **10 more models if Hubspot is enabled**
+    - **5 more models if Jira is enabled**
+    - **3 more models if Zendsk is enabled**
+- Updated `stg_rag_hubspot__owner` to correctly find columns from the owner source. Previously, this erroneously looked at the columns from the HubSpot `contact` table.
```

Comment:
This actually came from Levon's PR, should we add a link to it at the end in the normal format

```suggestion
- Updated `stg_rag_hubspot__owner` to correctly find columns from the owner source. Previously, this erroneously looked at the columns from the HubSpot `contact` table. ([#23](https://github.com/fivetran/dbt_unified_rag/pull/23))
```